recipes/mpcmd/meta.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: mpcmd
+  version: 0.0.5
+
+source:
+  url: https://files.pythonhosted.org/packages/bf/8c/07e22c42458f6dac37409979358b366cdf0c242bf648183cbdab6fba8eb5/mpcmd-0.0.5.tar.gz
+  sha256: b95b11b6465f73fcf4cbd095b873cac0140471b3ee1160ae980f4699bd08c302
+
+build:
```

Comment:
It looks like this package should be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python